sdk/monitor/Azure.Monitor.Ingestion/tests/IngestionClientTestEnvironment.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core.TestFramework;
+using Azure.Identity;
+
+namespace Azure.Monitor.Ingestion.Tests
+{
+    public class IngestionClientTestEnvironment : TestEnvironment
+    {
+        public string Endpoint => GetRecordedVariable("Ingestion_ENDPOINT");
+
+        public string DCRImmutableId => GetRecordedVariable("DCRImmutableId");
+
+        public string DCREndpoint => GetRecordedVariable("DCREndpoint");
+
+        public string WorkspaceId => GetRecordedVariable("WorkspaceId");
```

Comment:
It'd be good to switch the env var name strings to `Ingestion_WorkspaceId` in the future so you don't have conflicts when working across services.